src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs

```diff
@@ -300,6 +308,14 @@ public ConnectDialog(string appId, string appScope, Uri authorizeUrl, IAuthentic
                 .Select(project => project.Name)
                 .ToList();
 
+            if (!this.TeamProjects.Any())
+            {
+                reply.Text = Exceptions.NoTeamProjects;
```

Comment:
Same comment - why do we store normal text response in the Exceptions resource? Looks a little bit inconsistent.